index.bs

```diff
@@ -4262,11 +4262,15 @@ Revising a Recommendation: New Features</h5>
 	enables third parties to depend on a stable feature-set for [=Recommendations=] that do not advertise that ability,
 	as was the case for all [=Recommendations=] prior to the 2020 revision of this Process.
 
-	To make changes which introduce a new feature
-	to a [=Recommendation=] that does not [=allow new features=],
-	W3C <em class="rfc2119">must</em> create a new [=technical report=],
-	following the full process of <a href="#rec-advance">advancing a technical report to Recommendation</a>
+	Note: When a [=Recommendation=] does not [=allow new features=],
```

Comment:
```suggestion
	When a [=Recommendation=] does not [=allow new features=],
```
Drop "Note: " because in typical W3C contexts it implies a non-normative sentence/paragraph/section, and I don't think that was the intent of this change.

Reply:
No, that was actually intended. :) This paragraph is identifying an implication of existing normative text.